docs/src/tutorial/add-remove-nodes.md

```diff
@@ -0,0 +1,98 @@
+# Adding and Removing Nodes
+
+## What you will need
+- A bootstrapped cluster using Canonical K8s (See (getting-started.md)[Getting Started]
+- The `dns` and `network` components enabled on your cluster.
+
+### 1. Verify the Status Of Your Cluster
+
+Verify the status of your cluster with:
+
+```
+sudo k8s status
+```
+
+Make sure you can see `k8s is ready.` in the output of the command.
+
+### 2. Start a Multipass VM
+
+Our second node will live on a Multipass VM. Let's launch it:
+
+```
+multipass launch --name expert-elf
+```
+
+**Note**: This step can take a few minutes. It's normal and expected.
+
+### 3. Generate a worker node token from the control plane node
+
+In the shell connected to your control plane node, run the following command to get a token that'll be used by the other node to join the cluster.
```

Comment:
Let's say something like: From a shell in your bootstrapped (add link here again to bootstrapping your cluster) cluster we create a token that is used to allow other nodes to join the cluster.
Briefly, explain the difference between the worker and the control plane node and say in this example we are adding a worker node.

Reply:
The k8s status is to be run on the control plane node, my mistake.